internal/xds/translator/testdata/in/xds-ir/jwt-multi-route-multi-provider.yaml

```diff
@@ -21,12 +21,12 @@ http:
         - header: one-route-example-key1
           claim: claim.neteased.key
       - name: example2
-        issuer: https://www.two.example.com
+        issuer: http://www.two.example.com
         audiences:
         - one.foo.com
         - two.foo.com
         remoteJWKS:
-          uri: https://192.168.1.250:8080/jwt/public-key/jwks.json
+          uri: http://192.168.1.250:8080/jwt/public-key/jwks.json
```

Comment:
can we make the `localhost` endpoint the `http` endpoint and make this a `https` endpoint to mimic a real use